subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc

```diff
@@ -207,7 +207,7 @@ include::sample[dir="snippets/developingPlugins/publishingPlugins/groovy",files=
 include::sample[dir="snippets/developingPlugins/publishingPlugins/kotlin",files="build.gradle.kts[tags=local_repository]"]
 ====
 
-To use the repository in another build, you have to add it to <<plugins.adoc#sec:custom_plugin_repositories,the repositories of the `pluginManagement {}` block>> in you `settings.gradle(.kts)` file.
+To use the repository in another build, you have to add it to <<plugins.adoc#sec:custom_plugin_repositories,the repositories of the `pluginManagement {}` block>> in your `settings.gradle`(`.kts`) file.
```

Comment:
The typo is visible here: https://docs.gradle.org/current/userguide/publishing_gradle_plugins.html#custom-plugin-repositories